library-best-practices/JLBP-1.md

```diff
@@ -9,7 +9,8 @@ that library and that library's consumers.
 
 Some specific notes about minimizing dependencies:
 
-- Use the smallest scope possible. For example, auto-value doesn't
+- Use the smallest scope possible. For example, [`auto-value`]
```

Comment:
I'm not sure we need an inline link here. User testing in other docs has shown us that users tend to follow inline links and get lost and can't find their way back.